weed/compress/delta_binary_pack32.go

```diff
@@ -1,8 +1,8 @@
 package compress
 
 import (
-	"github.com/reducedb/encoding/cursor"
-	"github.com/reducedb/encoding/delta/bp32"
+	"github.com/dataence/encoding/cursor"
```

Comment:
is this related?

Reply:
The diff for fixing CI failed, the package changed its name in upstream, so `go get ./weed/...` command gets an error in CI. Maybe we should use glide to fetch dependence in CI.